<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ripes::MemoryViewerWidget</class>
 <widget class="QWidget" name="Ripes::MemoryViewerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="MemoryView" name="memoryView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="flowParentLayout" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MemoryView</class>
   <extends>QTableView</extends>
   <header>memoryview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
